indico/modules/events/templates/footer.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'footer.html' %}
 
 {% block footer_links_extra %}
-    {% if privacy_url %}
+    {% for privacy_url in privacy_urls  %}
```

Comment:
Maybe we could show a single link here and show a menu/dropdown-like widget if there are multiple choices?